src/api/admin/mermaid.py

```diff
@@ -651,7 +651,7 @@ class FishAttributeGroupingAdmin(FishAttributeAdmin):
     def region_list(self, obj):
         if not hasattr(obj, "regions"):
             return []
-        return ", ".join([r.name for r in obj.regions.order_by("name")])

```

Comment:
does the previous one just need to be changed to `obj.regions.all().order_by("name")`

Reply:
`obj.regions` is just a list of `UUID`s